src/database/migrations/20240429100117-create-user.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      username: {
+        type: Sequelize.STRING,
+      },
+      email: {
+        type: Sequelize.STRING,
+      },
+      password: {
+        type: Sequelize.STRING,
+      },
+      userRole: {
+        type: Sequelize.STRING,
+      },
+      verified: {
+        allowNull: false,
+        type: Sequelize.BOOLEAN,
+        defaultValue: false,
+      },
```

Comment:
revert to the old migration files and add a new one that adds a new column

Reply:
can you work on this, you don't have to create a new migration file, instead you need to alter table of users, as migration of users alread exists
https://sequelize.org/docs/v6/other-topics/query-interface/

Reply:
I created a new migration file for adding a new column, actually the migration file that I had had no effect as the table had already been created when we started using the same local database